### PR TITLE
Using KV api instead of vmkfstools command

### PR DESCRIPTION
**Pull request by pshahzeb**
Using kvEsx.get_info() instead of vmkfstools in get_vmdk_size_info()
to get volume utilization info. Using the api reduces the admin ls
command latency.

Testing: Manual testing using command /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
Output with earlier approach which used vmkfstools
```
Volume  Datastore      Created By VM  Created                   Attached To VM  Policy          Capacity  Used      Disk Format  Filesystem Type  Access      Attach As
------  -------------  -------------  ------------------------  --------------  --------------  --------  --------  -----------  ---------------  ----------  ----------------------
vol1    vsanDatastore  photon.rc (1)  Tue Feb 14 23:22:18 2017  detached        [VSAN default]  10.00GB   372.00MB  thin         ext4             read-write  independent_persistent
vol2    vsanDatastore  photon.rc (1)  Tue Feb 14 23:27:05 2017  detached        [VSAN default]  10.00MB   8.00MB    thin         ext4             read-write  independent_persistent
vol3    vsanDatastore  photon.rc (1)  Tue Feb 14 23:29:38 2017  detached        [VSAN default]  10.00MB   8.00MB    thin         ext4             read-write  independent_persistent
```

Output when kvEsx.get_info() api used.
```
olume  Datastore      Created By VM  Created                   Attached To VM  Policy          Capacity  Used   Disk Format  Filesystem Type  Access      Attach As
------  -------------  -------------  ------------------------  --------------  --------------  --------  -----  -----------  ---------------  ----------  ----------------------
vol1    vsanDatastore  photon.rc (1)  Tue Feb 14 23:22:18 2017  detached        [VSAN default]  10GB      372MB  thin         ext4             read-write  independent_persistent
vol2    vsanDatastore  photon.rc (1)  Tue Feb 14 23:27:05 2017  detached        [VSAN default]  10MB      8MB    thin         ext4             read-write  independent_persistent
vol3    vsanDatastore  photon.rc (1)  Tue Feb 14 23:29:38 2017  detached        [VSAN default]  10MB      8MB    thin         ext4             read-write  independent_persistent
```

Resolves: #39

### CODE REVIEW

**Comment by pshahzeb** [comment]
Updated.

File: esx_service/cli/vmdkops_admin.py [hunk 1]
```diff
@@ -619,24 +623,15 @@ def get_vmdk_size_info(path):
...
+    api is in human readable form
     """
     try:
-        cmd = "vmkfstools --extendedstatinfo {0}".format(path).split()
-        output = subprocess.check_output(cmd)
-        result = output.decode('utf-8')
-        lines = result.split('\n')
-        capacity_in_bytes = lines[0].split()[2]
-        used_in_bytes = lines[1].split()[2]
-        return {'capacity': human_readable(int(capacity_in_bytes)),
```
Comment (shuklanirdesh82) [inline esx_service/cli/vmdkops_admin.py]: can you please remove```human_readable``` method? as there will not be any consumer with your proposed changes.

File: esx_service/cli/vmdkops_admin.py [hunk 2]
```diff
@@ -619,24 +623,15 @@ def get_vmdk_size_info(path):
     Get the capacity and used space for a given VMDK given its absolute path.
     Values are returned as strings in human readable form (e.g. 10.00MB)
```
Comment (shuklanirdesh82) [inline esx_service/cli/vmdkops_admin.py]: nit: ```e.g. 10.00MB``` => e.g. 10MB to match with the kv.get_vol_info output

File: esx_service/cli/vmdkops_admin.py (hunk 1 above)
Comment (pshahzeb) [inline esx_service/cli/vmdkops_admin.py]: this method has references in ```generate_tenant_access_ls_rows``` method in the same file
The references are as below. 

```
        # p[auth_data_const.COL_MAX_VOLUME_SIZE] is max_volume_size in MB
        max_vol_size = UNSET if p.max_volume_size == 0 else human_readable(p.max_volume_size * MB)
        # p[auth_data_const.COL_USAGE_QUOTA] is total_size in MB
        total_size = UNSET if p.usage_quota == 0 else human_readable(p.usage_quota * MB)
```

Comment (shuklanirdesh82) [inline esx_service/cli/vmdkops_admin.py]: I see. Thanks for checking and providing the info!

File: esx_service/cli/vmdkops_admin.py (hunk 2 above)
Comment (pshahzeb) [inline esx_service/cli/vmdkops_admin.py]: Corrected.

**Comment by govint** [comment]
With this change #892 also will be resolved because vmdk_get_size_info()
was failing with datastore names that have spaces in them. I'll close #892
and open a new issue for supporting datastores with spaces in them.

**Pull request merged**